packages/react-components/src/components/AttachmentDownloadCards.test.tsx

```diff
@@ -19,17 +19,16 @@ describe('FileDownloadCards should be rendered properly', () => {
   });
 
   it('should render if it is FileSharingMetadata', async () => {
-    const metadata: AttachmentMetadata = {
+    const attachment: AttachmentMetadata = {
       name: 'MockAttachmentCard',
       extension: 'docx',
       url: 'mockUrl',
-      /* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */
       id: 'mockId'
     };
 
     const props = {
       userId: 'MockUserId',
-      fileMetadata: [metadata]
+      attachment: [attachment]
```

Comment:
strange to have an attachment props taking an array. Should this not be attachments?
Then, you can simply spread them into the function at line 39 more easily

Reply:
thanks updated